kg_microbe/transform_utils/ontology/ontology_transform.py

```diff
@@ -57,12 +66,12 @@
 from ..transform import Transform
 
 ONTOLOGIES = {
-    "ncbitaxon": "ncbitaxon.owl.gz",
-    "chebi": "chebi.owl.gz",
-    "envo": "envo.json",
-    "go": "go.json",
-    # "rhea": "rhea.json.gz", # Redundant to RheaMappingsTransform
-    "ec": "ec.json",
+    # "ncbitaxon": "ncbitaxon.owl.gz",
```

Comment:
let's just remove these commented out ontologies for testing and I think it's ready to merge!

Reply:
That was my mistake, ready to merge once you review!